docs/docs/architecture/client_broker_protocol.md

```diff
@@ -0,0 +1,1223 @@
+---
+layout: default
+title: Client/Broker Protocol
+parent: Architecture
+nav_order: 8
+---
+
+# BlazingMQ Client/Broker Protocol
+{: .no_toc }
+
+* toc
+{:toc}
```

Comment:
Are these tags working? I don't see expansion in preview https://github.com/bloomberg/blazingmq/blob/0d42983b42e6487741c774b7894be20c1186e905/docs/docs/architecture/client_broker_protocol.md#blazingmq-clientbroker-protocol

Reply:
They will work in Jekyll.